chain.schema.json

```diff
@@ -2,19 +2,21 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "codebase":{
     "git_repo": "https://github.com/osmosis-labs/osmosis",
-    "recommended_version": "v12.2.0",
+    "recommended_version": "v12.3.0",
     "compatible_versions": [
-      "v12.2.0"
+      "v12.3.0",
+      "v12.2.0",
+      "v12.2.1"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.2.0/osmosisd-12.2.0-linux-amd64?checksum=sha256:b4a0296b142b1a535f3116021d39660868a83fc66b290ab0891b06211f86fd31",
-      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.2.0/osmosisd-12.2.0-linux-arm64?checksum=sha256:84717e741b61ef3616fef403aae42067614e58a0208347cd642f6d03240b7778"
+      "linux/amd64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.3.0/osmosisd-12.3.0-linux-amd64?checksum=sha256:958210c919d13c281896fa9773c323c5534f0fa46d74807154f737609a00db70",
+      "linux/arm64": "https://github.com/osmosis-labs/osmosis/releases/download/v12.3.0/osmosisd-12.3.0-linux-arm64?checksum=sha256:a931618c8a839c30e5cecfd2a88055cda1d68cc68557fe3303fe14e2de3bef8f"
     },
     "cosmos_sdk_version": "0.45",
     "tendermint_version": "0.34",
     "cosmwasm_version": "0.28",
     "cosmwasm_enabled": true,
-    "ibc_go_version": "3.3.0",
+    "ibc_go_version": "3.3.1",
```

Comment:
v12.3.0 seems to still be on 3.3.0:
https://github.com/osmosis-labs/osmosis/blob/7e41ec9e69bce66f519ae715d2a8adf74e377263/go.mod#L11